Cargo.toml

```diff
@@ -146,5 +146,5 @@ bulletproofs = { git = "https://github.com/dalek-cryptography/bulletproofs", rev
 cpuid-bool = { git = "https://github.com/eranrund/RustCrypto-utils", rev = "74f8e04e9d18d93fc6d05c72756c236dc88daa19" }
 
 # We need to patch aes-gcm so we can make some fields/functions/structs pub in order to have a constant time decrypt
-aes-gcm = { git = "https://github.com/xoloki/AEADs", rev = "d1a8517d3dd867ed9c5794002add67992a42f6aa" }
```

Comment:
i cloned xoloki repo into mobilecoinofficial, this is not a code change in that repo

Reply:
This should also happen in `consensus/enclave/trusted/Cargo.toml` (and please remember to do it in uprev as well)